pkgs/os-specific/linux/kbd/default.nix

```diff
@@ -8,15 +8,24 @@ stdenv.mkDerivation rec {
     sha256 = "1vcl2791xshjdpi4w88iy87gkb7zv0dbvi83f98v30dvqc9mfl46";
   };
 
-  configureFlags = "--disable-nls";  
+  /* Get the dvorak programmer keymap (present in X but not in kbd) */
+  dvpSrc = fetchurl {
+    url = "http://kaufmann.no/downloads/linux/dvp-1_2_1.map.gz";
+    sha256 = "0e859211cfe16a18a3b9cbf2ca3e280a23a79b4e40b60d8d01d0fde7336b6d50";
+  };
+
+  configureFlags = "--disable-nls";
```

Comment:
On Fri, 30 Jan 2015 06:11:03 -0800
Peter Simons <notifications@github.com> wrote:

> Is `--disable-nls` related to the Dvorak map?

No it was already there. I put the `dvpSrc` before to keep all the
source url at the same place.

This look like some end of line space cleaning making this change as if
it was part of this patch.

-- 
Mayeu a.k.a Matthieu
http://6x9.fr
GPG: A016 F2D8 0472 1186 1B33  A419 22B1 0496 B00D A693